docs/forms.rst

```diff
@@ -1,3 +1,93 @@
-
 Forms
 =====
+
+``is_core.forms`` contains helpers for forms rutines. The most interesting class is ``SmartForm``.
+
+SmartForm
+---------
+A base class for forms. Image that you want to call a method when a form is constructed. A basic approach is
```

Comment:
Trochu bych to tu konkretizoval: 

Imagine you need to dynamically change behaviour of your form fields, e.g. make a field required when some condition holds. In pure Django, you would override the ``__init__`` method and change the field attributes there. However, that's pretty uncomfortable as you need to explicitly call ``super``. More importantly, the ``__init__`` method tends to grow and becoming unreadable for complex forms. With ``SmartForm``, you can define ``_init_fields`` method instead and put your field-related logic there. For example:

{tady ten příklad}

Even more readable way is to define the ``_init_{field_name}`` method for the individual fields thus removing the clumsy ``for`` cycle and ifs.

{příklad pro init_field_name}